rest_framework/utils/model_meta.py

```diff
@@ -146,7 +152,11 @@ def _get_reverse_relationships(opts):
         )
 
     # Deal with reverse many-to-many relationships.
-    for relation in opts.get_all_related_many_to_many_objects():
+    if django.VERSION < (1, 9):
```

Comment:
Yeah, I started so, but then decided to move it to the only place where it's used and there are already some compat things in the method. Do you what to move all compat-related things to `compat.py` or just the latest addition?